### ISSUE TITLE
Keep camera orientation when hopping worlds

### ISSUE THREAD

**Issue by bebeli555** [runelite/runelite#13969]
A plugin or a thing that would keep your camera orientation when hopping worlds would be very good. I find it to be one of the most annoying things when ur hopping worlds then it snaps ur camera back to the default position every time you hop. I think it already remembers the vertical orientation when hopping but not the horizontal. Basically its a bug in the default client but jagex will never fix it.

**Comment by Hydrox6** [comment]
Duplicate of #12193

**Comment by bebeli555** [comment]
Oh nice. I guess it could also be turned on as default as its kinda a bug and i dont think anyone would prefer to have it.
